test/e2e/tests/inputs/array.spec.ts

```diff
@@ -42,17 +42,17 @@ test(`file drop event should not propagate to dialog parent`, async ({
 
   // Open the dialog.
   await page.getByRole('button', {name: fileName}).click()
-  await expect(page.locator('#tree-editing-dialog')).toBeVisible()
+  await expect(page.getByRole('dialog')).toBeVisible()
```

Comment:
This test file has been updated to how it was before the array tree editing feature